package.json

```diff
@@ -251,7 +251,8 @@
     "@ledgerhq/hw-app-eth@npm:^6.39.0": "patch:@ledgerhq/hw-app-eth@npm%3A6.39.0#~/.yarn/patches/@ledgerhq-hw-app-eth-npm-6.39.0-866309bbbe.patch",
     "@ledgerhq/evm-tools@npm:^1.2.3": "patch:@ledgerhq/evm-tools@npm%3A1.2.3#~/.yarn/patches/@ledgerhq-evm-tools-npm-1.2.3-414f44baa9.patch",
     "cross-spawn@npm:^5.0.1": "^7.0.6",
-    "@solana/web3.js@npm:^1.95.0": "^1.95.8"
+    "@solana/web3.js@npm:^1.95.0": "^1.95.8",
+    "tslib": "~2.6.0"
```

Comment:
this `tslib` forced resolution has been added to fix this error that prevents the wallet unlock:

```
TypeError#1: Cannot assign to read only property 'Symbol(Symbol.iterator)' of object '[object Object]'
(anonymous) @ sentry-install.js:37776
sentry-install.js:37776   at Object.__generator (chrome-extension://bclblcklhpbhilglihnehjhmpfchmmpp/common-8.js:49554:131)
  at Mutex.<anonymous> (chrome-extension://bclblcklhpbhilglihnehjhmpfchmmpp/common-5.js:22679:28)
  at chrome-extension://bclblcklhpbhilglihnehjhmpfchmmpp/common-8.js:49548:41
  at new Promise (<anonymous>)
  at Object.__awaiter (chrome-extension://bclblcklhpbhilglihnehjhmpfchmmpp/common-8.js:49544:16)
  at Mutex.acquire (chrome-extension://bclblcklhpbhilglihnehjhmpfchmmpp/common-5.js:22676:24)
  at CurrencyRateController.updateExchangeRate (chrome-extension://bclblcklhpbhilglihnehjhmpfchmmpp/common-1.js:21928:42)
  at CurrencyRateController._executePoll (chrome-extension://bclblcklhpbhilglihnehjhmpfchmmpp/common-1.js:21994:16)
  at chrome-extension://bclblcklhpbhilglihnehjhmpfchmmpp/common-1.js:24712:24
  at sentryWrapped (chrome-extension://bclblcklhpbhilglihnehjhmpfchmmpp/scripts/sentry-install.js:21642:17)
  ```